plugins/flowdock/app/decorators/stage_decorator.rb

```diff
@@ -0,0 +1,21 @@
+Stage.class_eval do
```

Comment:
I know, but I like to see a class eval on this cases due to 2 reasons.
First, when I am re-opening a class that is not defined by me (on this case the plugin) I like to use ```class_eval``` as this will clearly state *this model is defined elsewhere*.
Second, if anything happens to the model (deleted/removed) ```class_eval``` will raise an error, which will allow me to detect that I need to either rename the variable or the code no longer makes so exist.